lib/SmileIdentityCore.php

```diff
@@ -215,23 +215,25 @@ public function get_job_status($partner_params, $options)
 
     /***
      *  Will query the backend for web session token with a specific timestamp
-     * @param timestamp the timestamp to generate the token from
+     * @param timestamp the iso 8601 date/time format to generate the token from
      * @param user_id
      * @param job_id
      * @param product_type - Literal value of any of the 6 product type options
      * @return array
      * @throws GuzzleException
      */
```

Comment:
Kindly update the documentation to reflect the method's new signature/definition (ie. parameters, etc.)